BoxBrowseSDK/BoxBrowseSDK.xcodeproj/project.pbxproj

```diff
@@ -224,15 +224,15 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				C1E7651D66F260349D91B959 /* libPods.a in Frameworks */,
+				A3DAF34160188D517F8283B9 /* Pods.framework in Frameworks */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 		15146EB81AC20B78000BA380 /* Frameworks */ = {
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				1C6E5394A7F19AE4673FFCCC /* libPods.a in Frameworks */,
+				1C6E5394A7F19AE4673FFCCC /* (null) in Frameworks */,
```

Comment:
It's a cocoapods issue:
https://github.com/CocoaPods/Xcodeproj/issues/248

I'm not sure it's worth the risk to attempt the suggested fix though.